Rebranding

IvanKiral opened the pull request:
### Motivation

Which issue does this fix? Fixes #`issue number`

Migration and rebranding

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

Simply007 reviewed (changes requested):
Notes:
- [x] package.json - rename name of the packages (remove `kontent-` from package name)
- [ ] Remove GT-Walfheim and use OS font from the guide
- [ ] (optional) take a look to packages (try at least minor versions bump)

Great work with the Store rewrite! 👍🏼

Simply007 commented:
Upgrade will be done in #102

Simply007 reviewed (changes requested):
- [ ] Let's unify the configuration page with new visuals (logo colors)
- [ ] Add redirects to netlify: https://github.com/kontent-ai/sample-app-react/blob/master/netlify.toml
- [ ] Try to unify font to WorkSans - if something does not look OK - try to adjust font-weight
- [x] Coffees filter does not work
- [ ] Brewers filter is missing
- [ ] I don't se subtitle on Home hero banner, Titles on Latest articles
- [ ] Rich text resolution for twitter widget and video is not resolved `~/en-us/articles/cf106f4e-30a4-42ef-b313-b8ea3fd3e5c5`

Simply007 reviewed (changes requested):
If i go to some page and back - i don't see the cafes:

![recording](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiY2I2ZDMzOTQtMWVlOS00OTFkLTk0OWYtOGU5ZDQ3NDBmZjM2IiwiZGVwbG95X2lkIjoiNjJmNGI0YWNjYTI1MzIwMDA5ZDU1NjAzIiwiaWQiOiIwNGIyMjg0Yy0zMDk2LTRlN2ItYjVmZi00ODczMGU2YzFjMzcifQ==)

If I go to https://deploy-preview-101--kontent-sample-app-vue.netlify.app/es-ES/articles/117cdfae-52cf-4885-b271-66aef6825612 (ES language) the link in text does not preserve nor load currect language varian of the coffee (link is `https://deploy-preview-101--kontent-sample-app-vue.netlify.app/coffees/kenya-gakuyuni-aa`)

IvanKiral commented:
I agree with the first one, there was one additional condition which made this error. The second one might be due to fallbacking into English language due to not published content in ES language.

Simply007 commented:
> I agree with the first one, there was one additional condition which made this error. The second one might be due to fallbacking into English language due to not published content in ES language.

In that case, there should be ˙https://deploy-preview-101--kontent-sample-app-vue.netlify.app/en-US/coffees/kenya-gakuyuni-aa˙ (with language codename right). The thing is we don't want to have the same content on a different URLs. Not sure about the fallback - If we want to change the language prefix - but is should be present.

Simply007 reviewed (changes requested):
Everything, but language behavior is fine.

- [x] lowercase language prefixes
- [ ] default route should enforce prefix `/articles` -> `/en-us/articles` 
- [ ] If a fallback is used enforce the fallback language right to the prefix and basically change it
- [ ] Describe the behavior in the README -> at least brief - to be ready

Simply007 reviewed (changes requested):
- [ ] Homepage link for About us has a "More Articles" label.

Haven't found anything more.

Simply007 reviewed (changes requested):
Update the readme to reflect the current state.

IvanKiral commented:
I have updated the readme. Added  a new paragraph about data fetching with an example, fixed grammar mistakes, and adjusted the structure of the whole readme

Simply007 commented on src/components/Admin/Configuration.vue:
```diff
@@ -189,9 +184,9 @@ export default {
     redirectToHome(newProjectId) {
```
I didn't get to be redirected after generating the project.

Simply007 replied:
Feel free to submit it separately - I don't think your changes broke it.

Simply007 reviewed (commented):
Great work on README.

I have one comment about configuration + I might change the name of the kk-logo.svg to something more accurate.
